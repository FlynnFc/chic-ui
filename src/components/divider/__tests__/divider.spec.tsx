import Divider from '..';
import { render } from '../../../test/utils';

describe('Divider Component', () => {
  it('Renders Correctly', () => {
    const DividerGroup = render(
      <>
        <div>
          <Divider />
          <Divider type="danger" />
          <Divider type="info" />
          <Divider type="light" />
          <Divider type="secondary" />
          <Divider type="success" />
          <Divider type="warning" />
        </div>
        <div>
          <Divider dashed />
          <Divider type="danger" dashed />
          <Divider type="info" dashed />
          <Divider type="light" dashed />
          <Divider type="secondary" dashed />
          <Divider type="success" dashed />
          <Divider type="warning" dashed />
        </div>
      </>
    );
    expect(DividerGroup).toMatchSnapshot();
  });
});
