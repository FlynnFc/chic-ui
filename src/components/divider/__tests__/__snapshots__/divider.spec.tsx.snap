// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component Renders Correctly 1`] = `
Array [
  <div>
    <hr
      className="sc-bdfBQB llzgni"
      type="secondary"
    />
    <hr
      className="sc-bdfBQB cMDGnu"
      type="danger"
    />
    <hr
      className="sc-bdfBQB iAuQuU"
      type="info"
    />
    <hr
      className="sc-bdfBQB hRmSIM"
      type="light"
    />
    <hr
      className="sc-bdfBQB llzgni"
      type="secondary"
    />
    <hr
      className="sc-bdfBQB eHGtSr"
      type="success"
    />
    <hr
      className="sc-bdfBQB jtxyWa"
      type="warning"
    />
  </div>,
  <div>
    <hr
      className="sc-bdfBQB fPkkHA"
      type="secondary"
    />
    <hr
      className="sc-bdfBQB jklyIY"
      type="danger"
    />
    <hr
      className="sc-bdfBQB hfnQgK"
      type="info"
    />
    <hr
      className="sc-bdfBQB hwqpRa"
      type="light"
    />
    <hr
      className="sc-bdfBQB fPkkHA"
      type="secondary"
    />
    <hr
      className="sc-bdfBQB cHICOZ"
      type="success"
    />
    <hr
      className="sc-bdfBQB dodOXU"
      type="warning"
    />
  </div>,
]
`;
